packages/amplify-category-storage/src/__tests__/provider-utils/awscloudformation/service-walkthroughs/s3-walkthrough.test.ts

```diff
@@ -142,7 +142,7 @@ describe('add s3 walkthrough tests', () => {
       .mockResolvedValueOnce([S3PermissionType.CREATE_AND_UPDATE, S3PermissionType.READ, S3PermissionType.DELETE]) // What kind of permissions (Auth)
   
 
-    prompter.confirmContinue = jest
+    prompter.yesOrNo = jest
```

Comment:
we already fixed this right in earlier PR?

Reply:
This was missed out in the earlier PR